README-Docker.md

```diff
@@ -0,0 +1,68 @@
+# Docker
```

Comment:
I think we can put this instruction under the quick start section of the main `README.md`?

Reply:
@bwalsh, I saw you inserted this into README.md. Could you delete this?

Reply:
done